js/src/redux/reducers/settings.js

```diff
@@ -0,0 +1,17 @@
+import { SET_SETTINGS } from "../actions/settings";
+
+/**
+ * A reducer for settings.
+ *
+ * @param {Object} state The current state of the object.
+ * @param {Object} action The current action received.
+ *
+ * @returns {Object} The state.
+ */
+export default function settingsReducer( state = {}, action ) {
```

Comment:
Settings is a very generic name for what is currently saved in it. What was the goal of `settings` in state?